### PR TITLE
Sending and saving of nodule properties on Annotate screen

### PR DESCRIPTION
**Pull request by Serhiy-Shekhovtsov**
Fixed few bugs of Annotate screen. Fixed sending of the Nodule data to backend. Added fields for storing `probability_concerning` and `note`. Includes fix of #285

## CLA
- [x] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

File: interface/backend/api/serializers.py [hunk 1]
```diff
@@ -95,6 +95,9 @@ class Meta:
     def create(self, validated_data):
         return Nodule.objects.get_or_create(candidate=validated_data['candidate'])
 
+    def update(self, instance, validated_data):
+        return super().update(instance, validated_data)
```
Comment (WGierke) [inline interface/backend/api/serializers.py]: Why is this necessary? It doesn't overwrite the update method inherited from the parent class, does it?

Comment (Serhiy-Shekhovtsov) [reply interface/backend/api/serializers.py]: Yes, put it while troubleshooting an other issue and forgot to remove. Thanks for noting!
Will fix the PR tomorrow.
